geotrek/settings/tests.py

```diff
@@ -43,3 +43,11 @@ def __getitem__(self, item):
 
 
 MIGRATION_MODULES = DisableMigrations()
+
+SURICATE_SETTINGS = {
+    'URL': 'http://sentinelles-preprod.sportsdenature.fr/rest/suricate/wsstandard/',
+    'ID_ORIGIN': 'geotrek',
+    'PRIVATE_KEY_CLIENT_SERVER': '',
+    'PRIVATE_KEY_SERVER_CLIENT': '',
+    'ID_USER': 'XXX123',
+}
```

Comment:
as you defined basic settings in base.py and mocking tests, do you need to keep this here ?